seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisSink.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.flink.types.Row;
 import org.apache.flink.util.Preconditions;
 
+import javax.annotation.Nullable;
```

Comment:
Does the `javax` package class import after the `java`?

Reply:
Hi @SteNicholas , I believe it’s not big deal, also you are welcome to create a PR to add a rule into code style if you are free

Reply:
Hi @SteNicholas, thanks for you comment, I agree with you, and I just create a PR to add a rule into Code Style for the order of improts to further ensure uniformity of code style across each connector.